src/nvim/ex_docmd.c

```diff
@@ -8761,7 +8761,10 @@ makeopens (
      * Do this before restoring the view, so that the topline and the
      * cursor can be set.  This is done again below.
      */
-    if (put_line(fd, "set winheight=1 winwidth=1") == FAIL)
+    if (put_line(
+          fd,
+          "set winminheight=1 winminwidth=1 winheight=1 winwidth=1"
+        ) == FAIL)
```

Comment:
the linter wants it like this:

```c
if (put_line(fd,
             "set ..."
```

But the arguments fit on one line I think, so that would be best instead.